kyuubi-server/pom.xml

```diff
@@ -221,6 +221,16 @@
             <artifactId>jersey-media-multipart</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jdk8</artifactId>
```

Comment:
does it compatiable with JDK11 and 17 ?

Reply:
Yes, the data type we currently declare in the Trino HTTP req/res DTO are of java8 DataType. [Java 8 Datatypes](https://github.com/FasterXML/jackson-modules-java8/blob/2.15/datatypes): support for other new Java 8 datatypes outside of date/time: most notably Optional, OptionalLong, OptionalDouble.